docs/ClusterSetup.md

```diff
@@ -133,11 +133,11 @@ for the first time. The following command will prepare both the BookKeeper
 as well as the Pulsar metadata.
 
 ```shell
-$ bin/pulsar --cluster us-west \
-             --zookeeper zk1.us-west.example.com:2181 \
-             --global-zookeeper zk1.us-west.example.com:2184 \
-             --service-url http://pulsar.us-west.example.com:8080/ \
-						 --service-url-tls https://pulsar.us-west.example.com:8443/
+$ bin/pulsar initialize-cluster-metadata --cluster us-west \
+                                         --zookeeper zk1.us-west.example.com:2181 \
+                                         --global-zookeeper zk1.us-west.example.com:2184 \
+                                         --service-url http://pulsar.us-west.example.com:8080/ \
+						                             --service-url-tls https://pulsar.us-west.example.com:8443/
```

Comment:
Corrected in https://github.com/yahoo/pulsar/pull/21/commits/a6df34360fbf45c3948ae83cea8c754e399e0368